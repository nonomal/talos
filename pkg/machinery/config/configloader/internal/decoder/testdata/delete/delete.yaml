apiVersion: v1alpha1
kind: SideroLinkConfig
$patch: delete
---
apiVersion: v1alpha1
kind: ExtensionServiceConfig
name: foo
configFiles:
- content: hello
  $patch: delete
- content: hello2
  mountPath: /etc/foo2
---
version: v1alpha1
machine:
  hostname:
    $patch: delete
  network:
  - interface: eth0
    $patch: delete
  - interface: eth1
    addresses: [10.3.5.5/32]
  - interface: eth0
    dhcp6: true
