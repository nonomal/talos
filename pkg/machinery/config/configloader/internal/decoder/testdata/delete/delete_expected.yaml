apiVersion: v1alpha1
kind: ExtensionServiceConfig
name: foo
configFiles:
    - content: hello2
      mountPath: /etc/foo2
---
version: v1alpha1
machine:
    network:
        - interface: eth1
          addresses: [10.3.5.5/32]
        - interface: eth0
          dhcp6: true
